src/config/plugin.ts

```diff
@@ -79,6 +79,8 @@ export class Plugin implements IPlugin {
 
   root!: string
 
+  pluginAlias!: string
```

Comment:
Add pluginAlias property

Reply:
nitpick: I'd prefer `alias` over `pluginAlias` since it's already on the `Plugin` class